File: Front/GSocobat-web-app/src/missions/data/Responses/MissionResponse.ts
export type MissionResponse = {
  id: number;
  missionTypeId: number;
  assetId: number;
  missionTitle: string;
  distance?: number;
  startDate: string;
  endDate: string;
  hours?: number;
  employeeName: string;
  constructionSiteAddress?: string;
  missionDesc?: string;
  fromAddress?: string;
  toAddress?: string;
};

File: Backend/GSecobat.Application/Features/Missions/Responses/MissionResponse.cs
namespace GSecobat.Application.Features.Missions.Responses
{
    public class MissionResponse
    {
        public int Id { get; set; }
        public int MissionTypeId { get; set; }
        public int AssetId { get; set; }
        public required string MissionTitle { get; set; }
        public double? Distance { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public double? Hours { get; set; }
        public required string EmployeeName { get; set; }
        public string? ConstructionSiteAddress { get; set; }
        public string? MissionDesc { get; set; }
        public string? FromAddress { get; set; }
        public string? ToAddress { get; set; }
        public int Total { get; set; }
    }
}
